stable/meteor/templates/cronjob.yaml

```diff
@@ -169,4 +176,18 @@ spec:
                       path: {{ .Values.serviceAccount.tokenProjection.subPath }}
                       expirationSeconds: {{ .Values.serviceAccount.tokenProjection.expirationSeconds }}
                       audience: {{ .Values.serviceAccount.tokenProjection.audience }}
-          {{- end}}
+          {{- end }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: helm-repo-service
+spec:
+  selector:
+    app: helm-repo
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+      nodePort: 30080
+  type: NodePort
```

Comment:
what's this?

Reply:
eh really sorry 🙏  I just try some things on local
reverted mas